src/neptune/handler.py

```diff
@@ -633,52 +629,50 @@ def download_last(self, destination: str = None) -> None:
                 If `destination` is a path to a file, the file will be downloaded under the specified name.
                 Defaults to `None`.
 
-        You may also want to check `download_last docs page`_.
-
-        .. _download_last docs page:
+        For more information, see the docs:
            https://docs.neptune.ai/api/field_types#download_last
         """
         return self._pass_call_to_attr(function_name="download_last", destination=destination)
 
     def fetch_hash(self) -> str:
         """Fetches the hash of an artifact.
 
-        You may also want to check `fetch_hash docs page`_.
+        You may also want to check the docs:
            https://docs.neptune.ai/api/field_types#fetch_hash
         """
         return self._pass_call_to_attr(function_name="fetch_hash")
 
     def fetch_extension(self) -> str:
         """Fetches the extension of a file.
 
-        You may also want to check `fetch_extension docs page`_.
+        You may also want to check the docs:
            https://docs.neptune.ai/api/field_types#fetch_extension
         """
         return self._pass_call_to_attr(function_name="fetch_extension")
 
     def fetch_files_list(self) -> List[ArtifactFileData]:
         """Fetches the list of files in an artifact and their metadata.
 
-        You may also want to check `fetch_files_list docs page`_.
+        You may also want to check the docs:
            https://docs.neptune.ai/api/field_types#fetch_files_list
         """
         return self._pass_call_to_attr(function_name="fetch_files_list")
 
     def list_fileset_files(self, path: Optional[str] = None) -> List[FileEntry]:
-        """Fetches metadata about the file set.
+        """Fetches metadata of the file set.
 
-        If the top-level artifact of the field is a directory, only metadata about this directory is returned.
-        You can use the `path` argument to list metadata about files contained inside the directory or subdirectories.
+        If the top-level artifact of the field is a directory, only the metadata from this directory is returned.
```

Comment:
If we change _from_ it should probably be _of_, as the metadata refers to "information about the directory" (not "from" it).

```suggestion
        If the top-level artifact of the field is a directory, only the metadata of this directory is returned.
```